featuretools/tests/entityset_tests/test_entity.py

```diff
@@ -127,3 +127,19 @@ def test_delete_variables(es):
     for var in to_delete:
         assert var not in variable_names
         assert var not in entity.df
+
+
+def test_variable_types_copy():
```

Comment:
Using copy is how we fixed the problem but I think the test name should describe the problem

Reply:
Fixed